hm6_2.py

```diff
@@ -0,0 +1,10 @@
+""" Task 2: "I love arrays they are my favorite"
+
+=> ["I", "love", "arrays", "they", "are", "my", "favorite"]
+
+"""
+
+TXT = "I love arrays they are my favorite"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```

Comment:
here as well

Reply:
Understand your message, will fix, thank you.